Min node version must be at least v10

Readme file specifies 
[Node.js](https://nodejs.org/en/download/) (8.9.4 or newer, we recommend using the current Long Term Stable version),

But min working version is v.10.

Attempt to run with in node v 8.12.0 resulted in error:

> C:\ru102js\node_modules\update-notifier\index.js:58
>                         } catch {
>                                 ^
> 
> SyntaxError: Unexpected token {
>     at createScript (vm.js:80:10)
>     at Object.runInThisContext (vm.js:139:10)
>     at Module._compile (module.js:617:28)
>     at Object.Module._extensions..js (module.js:664:10)
>     at Module.load (module.js:566:32)
>     at tryModuleLoad (module.js:506:12)
>     at Function.Module._load (module.js:498:3)
>     at Module.require (module.js:597:17)
>     at require (internal/module.js:11:18)
>     at Object.<anonymous> (C:\ru102js\node_modules\nodemon\bin\nodemon.js:15:3)
> npm ERR! code ELIFECYCLE
> npm ERR! errno 1
> npm ERR! redisolar@1.0.0 dev: `nodemon`
> npm ERR! Exit status 1
> npm ERR!
> npm ERR! Failed at the redisolar@1.0.0 dev script.
> npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
>

--- a/README.md
+++ b/README.md
@@ -12,7 +12,7 @@ Solutions to the course programming challenges can be found on the `solutions` b
 
 In order to start and run this application, you will need:
 
-- [Node.js](https://nodejs.org/en/download/) (8.9.4 or newer, we recommend using the current Long Term Stable version)
+- [Node.js](https://nodejs.org/en/download/) (We recommend using the current Long Term Stable version and have tested with version 18.4.2).
 - npm (installed with Node.js).
 - Access to a local or remote installation of [Redis Stack](https://redis.io/docs/stack/get-started/install/) - local preferred, we provide a Docker Compose file in this repo, read on for details.
 

